blockchain/pkg/client.go

```diff
@@ -0,0 +1,31 @@
+package pkg
```

Comment:
should it be `pkg`?

Reply:
It replicates the same structure as other modules in the workspace. Should we remove the `/pkg` nesting layer and put client, `pallets`, and `ddcprimitives` modules into the module root for `package blockchain`?

Reply:
i think it make sense. Otherwise you will have `pkg` in your import path in storage node or wherever it's used, right? I've never saw smth like import 'net/http/pkg'

Reply:
As far as I understand it tries to follow the https://github.com/golang-standards/project-layout, but [/pkg](https://github.com/golang-standards/project-layout/tree/master/pkg) there is a place to put libraries, not for internal structure of the library as we have here.

Reply:
Done https://github.com/Cerebellum-Network/cere-ddc-sdk-go/pull/70/commits/b50444f3d932f28624bcac3ddd39e5f9bcb18204.